resourcemonitor/resourcemonitor.go

```diff
@@ -602,6 +615,72 @@ func (monitor *ResourceMonitor) processAlerts() {
 	}
 }
 
+func (xen *xenSystemUsage) RunCmd() {
```

Comment:
I don't see where this function is actually called.

Reply:
Done. Sorry forgot to add